requirements.txt

```diff
@@ -24,12 +24,12 @@ dealer
 flask-collect
 flake8
 passlib
-twill
+https://github.com/alexandre/twill/tarball/master
```

Comment:
@rochacbruno ,

The author of the project twill (C. Titus Brown) sent me a answer: https://github.com/ctb/twill

I've sent another email asking him about plans to keep a python3 version on Pypi or a py2-py3 hybrid version.